README.md

```diff
@@ -11,7 +11,7 @@ You can install the PHP SDK with Composer, either run `composer require microsof
 ```
 {
     "require": {
-        "microsoft/microsoft-graph": "^1.35.0"
+        "microsoft/microsoft-graph": "^1.35.1"
```

Comment:
Seems like more of a patch than a minor upgrade to me since new metadata only updates comments

Reply:
Recently merged [PR](https://github.com/microsoftgraph/msgraph-sdk-php/pull/561) updates GraphConstants